docs/introduction/zns.md

```diff
@@ -47,91 +47,89 @@ title="Conventional SSDs and ZNS SSDs internal data placement"/>
-reported namespace capacity if the namespace contains zones with a zone capacity
-lower than the zone size.
+total number of logical blocks defined by the zones. In other words, if the
+namespace contains zones with zone capacity less than the zone size, the
+reported capacity includes unusable logical blocks (falling b/w ZCAP & ZSZE) and
+the usable capacity of the namespace is equal to the sum of all zones' capacities.
+In this case, the usable capacity available is smaller than the reported namespace
+capacity.
 :::
 
 ### Active Zones
 
 A controller implementation typically requires the allocation of internal
-resources (e.g. a write buffer) to execute write operations into zones.
-Therefore, limitations on the total amount of resources available to the
-controller may imply a limit on the total number of zones that can be
-simultaneously in the implicit open or explicit open conditions. This potential
-limit on the maximum number of open zones is similarly defined in the ZNS, ZBC,
-and ZAC standards.
-
-The ZNS specification however defines an additional limit on the number of zones
-that can be in the implicit open, explicit open or closed conditions. Any zone
-with such condition is defined as an active zone and correspond to any zone that
-is being written or that has been only partially written. A ZNS SSD may impose a
-limit on the maximum number of zones that can be active. This limit is always
-equal or larger than the limit on the maximum number of open zones.
+resources (e.g. write buffers) to execute write operations into zones.
+Limitations on the total amount of resources available in the controller may
+imply a limit on the total number of zones that can simultaneously be in the
+implicit open or explicit open states. This potential limit on the maximum
+number of open zones is similarly defined in the ZNS, ZBC and ZAC standards.
+
+The ZNS specification defines an additional limit on the number of zones
+that can be in the implicit open, explicit open or closed states. Zones mapped
+to these states are defined as active zones and correspond to zones that are being
+written to (partially written). A ZNS SSD may impose a limit on the maximum number
+of active zones. This limit is always equal to or greater than the maximum allowed
+open zones.
 
 This new limit imposes new constraints on user applications. While the maximum
 number of open zones of a namespace only limits the number of zones that an
 application can simultaneously write, the maximum number of active zones imposes
-a limit on the number of zones that an application can choose for storing data.
+a limit on the number of zones that an application can choose from for writing.
 If the maximum number of active zones is reached, the application must either
-reset or finish some active zones before being able to chose other zones for
-storing data.
+Reset or Finish at-least one active zone to be able to choose any other zone for
+writing (make the zone active).
```

Comment:
Remove the text in parenthesis. It is confusing.

Reply:
Updated